extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -28,6 +28,8 @@ export const CONSENT_ITEM_STATE = {
   UNKNOWN: 0,
   GRANTED: 1,
   REJECTED: 2,
+  DISMISSED: 3,
```

Comment:
I still think UI state and consent state should be separate. something like `UI_STATE = {pending, active, complete}` and `CONSENT_STATE = {unknown, granted, rejected}` I know you would have to refactor so up to you.